charts/keycloak/templates/poddisruptionbudget.yaml

```
@@ -1,5 +1,5 @@
 {{- if .Values.podDisruptionBudget -}}
-{{- if semverCompare ">=1.21.0" .Capabilities.KubeVersion.GitVersion }}
+{{- if .Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" }}
```

afaik the check needs be on the API not directly on the "API-Object"
```suggestion
{{- if .Capabilities.APIVersions.Has "policy/v1" }}
```

---

Hi @grieshaber,
I thought that since we are modifying a podDisruptionBudget template, we could directly specify the resource we are working on.
I commited the suggested change. Need to test it.
Kind regards.

---

Thanks @dides82 for the quick change! 
iirc the `.Capabilities.APIVersions.Has` checks against available APIVersions in the cluster. And the available API Version for the PDBs are `policy/v1` or `policy/v1beta1`.